UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -70,7 +77,13 @@ private void LayingDownLogic(bool forceState = false)
 
 		private void UpLogic(bool forceState = false)
 		{
-			if (forceState) sprites.localRotation = standingUp;
+			if (forceState)
```

Comment:
```suggestion
			if (forceState && sprites != null)
```
combine the conditions, no need to make another level of indentation.